vignettes/protein_structure_workflow.Rmd

```diff
@@ -0,0 +1,457 @@
+  scale_y_continuous(breaks = integer_breaks()) +
+  scale_x_continuous(breaks = integer_breaks()) +
+  labs(title = "Structure contact map 2HWG") +
+  facet_wrap(~chain_combinations, scale = "free") +
+  theme_bw()
+```
+The contact map for each structure provided as input will be saved as a separate element of a list, which is returned as the output of the `create_structure_contact_map()` function. These elements can be easily accessed using two square brackets (`[[]]`) right behind the name of the list. The brackets should either contain the name of the element or the index. You can also access each element in the list in a loop using the `map()` function from the `purrr` package.
+
+Contact maps can be plotted using the `ggplot2` package if they are not too large. Contact maps are based on positions of atoms or residues in the structure. While atom IDs are unique, residues are only unique within a certain chain. That means residue position 1 is likely present in each of the chains. To correctly display contact map plots for residues they should be faceted by the unique combinations of chains. If e.g. the x-axis displays residues from chain A and the y-axis residues of chain B this should be displayed in one specific facet. 
+
+_Note: For the plot we actually recommend using the data base defined chain and residue identifiers in contrast to the author defined ones that were used as input to the `create_structure_contact_map()` function. The reason is that in the data base definitions each molecular entity has their own chain. If a certain protein is associated with a metal ion, these might be stored in the same chain in the author defined identifiers but separated into different chains for the data base identifiers. The problem is that in the author defined identifiers the residue numbers might not be consecutive and there could be a large gap in the numbering between the protein and the metal residue._
+
+The two contact maps that were generated based on significant peptides reveal some interesting information about the positioning of the peptides. 
+
+The "1ZMR" structure contains only one protein chain (A). In the contact map of significant peptides that are all located in chain A in relation to all other residues in chain A we can see that even though some of the peptides are quite far apart in the sequence some are within 10 Angstrom distance in 3D space. Chain B and chain D are a calcium atom respectively. In case of chain B the atom is actually in close proximity to one of the significant peptides. Chain E stores information about water molecules in the structure. The contact map is not very informative in this case since the y-axis residue does not indicate any specific relationship of the water molecules with each other. Here it would be interesting to look at the distances of the water molecules to the peptide. If there are no water molecules in close proximity this might indicate that the peptide is not surface exposed.
```

Comment:
```suggestion
The "1ZMR" structure contains only one protein chain (A). In the contact map of significant peptides that are all located in chain A in relation to all other residues in chain A we can see that even though some of the peptides are quite far apart in the sequence, some are within 10 Angstrom distance in 3D space. Chain B and chain D are a calcium atom, respectively. In case of chain B the atom is in close proximity to one of the significant peptides. Chain E stores information about water molecules in the structure. The contact map is not very informative in this case, since the y-axis residue does not indicate any specific relationship of the water molecules with each other. Here it would be interesting to look at the distances of the water molecules to the peptide. If there are no water molecules in close proximity, this might indicate that the peptide is not surface exposed.
```